lib/controller.dart

```diff
@@ -341,13 +356,15 @@ void _computeChartBucketsAndRender() {
         var mapComparisonValues = Map<String, List<num>>();
         for (var field in chart.fields) {
           var regionName = field.field.value.toString();
+          var normalisationValue =
+              _dataNormalisationEnabled ? 100 : _filterValuesCount;
           mapValues[regionName] = [
             field.bucket[0],
-            field.bucket[0] / _filterValuesCount,
+            field.bucket[0] / normalisationValue,
           ];
           mapComparisonValues[regionName] = [
             field.bucket[1],
-            field.bucket[1] / _comparisonFilterValuesCount
+            field.bucket[1] / normalisationValue
```

Comment:
I think you're missing a statement similar to that on lines 359-360 for computing the normalised value for `_comparisonFilterValuesCount`?

Reply:
Good catch, my bad. Fixed